assets/extensions/all-extensions.js

```diff
@@ -1,73 +1,292 @@
 /**
- * WordPress dependencies
+ * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { keyBy } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Card from './card';
+import { Grid, Col } from './grid';
 
-const AllExtensions = ( { extensions } ) => {
-	const featuredExtensions = extensions.filter(
-		( extension ) => extension.is_featured
-	);
+// TODO: Get from API.
+const extensionsSkeleton = [
+	{
+		key: 'featured',
+		columns: 12,
+		type: 'featured-list',
+		title: 'Featured',
+		items: [
+			{
+				key: 'sensei-wc-paid-courses',
```

Comment:
All these `keys` can be a little boring because it's only for the React keys, but we don't have other information we could use for that, and the `index` could cause troubles for us in the future depending on how we enhance the features.

Reply:
Could be worth to move the sample to a separate file to keep the diffs clearer.

Reply:
Moved here: https://github.com/Automattic/sensei/commit/cfe89eb3dcc64b975500722be9fd68b09563672d